libs/ui-lib-tests/cypress/views/customManifestsPage.ts

```diff
@@ -0,0 +1,29 @@
+export const customManifestsPage = {
+  getFileName: (index: number) => {
+    return cy.findByTestId(`filename-${index}`);
+  },
+  getFileNameError: () => {
+    return cy.get('div.pf-c-form__helper-text.pf-m-error');
+  },
+  getStartFromScratch: () => {
+    return cy.get('.pf-c-file-upload .pf-c-empty-state__secondary > button');
+  },
+  fileUpload: (index: number) => {
+    return cy.findByTestId(`yamlContent-${index}`).get('input[type="file"]');
```

Comment:
Just one last thing: if you want to search for an element within the scope of another element, you need to use `find` instead of `get`. (See https://docs.cypress.io/api/commands/get#Get-vs-Find)